app/scripts/services/hpa.js

```diff
@@ -146,15 +170,19 @@ angular.module("openshiftConsole")
         return $q.when([]);
       }
       return MetricsService.isAvailable().then(function(metricsAvailable) {
+        var isV2HPA = hasV2HPAScalerWarning(scaleTarget, hpaResources);
```

Comment:
Why not just use `hasV2HPAAnnotations` here and remove `hasV2HPAScalerWarning`?

Reply:
Yeah that makes sense.